### ISSUE TITLE
Open Project in File Manager

### ISSUE THREAD

**Issue by brannondorsey** [olab-io/ofSketch#105]
Should we have a browser button to open a project (or at least its `data/` folder) in the default file manager. Would this kind of functionality be useful or would it encourage "bad" behavior.

**Comment by bakercp** [comment]
Yes, we should have this.  Maybe we could gamify ofSketch and only give people options like this after they have used it successfully for some amount of time ...  The goal of the "ofSketch: The Game" becomes to "escape from ofSketch" ;)
